test/commands/iteration/create_test.rb

```diff
@@ -199,7 +199,7 @@ class Iteration::CreateTest < ActiveSupport::TestCase
       submission = create :submission, solution: solution
       Iteration::Create.(solution, submission)
 
-      assert_equal Badges::DieUnendlicheGeschichteBadge, user.reload.badges.first.class
+      assert_includes user.reload.badges.map(&:class), Badges::DieUnendlicheGeschichteBadge
```

Comment:
I've changed the assertion slightly to make it clear that we don't care for the order of the badges. 
This was, we could refactor the order of the award jobs being run without the test breaking.